.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,28 @@
+# Workflow to run integration tests
+# Checks that check_new_repo workflow works as expected
+
+name: Integration Tests
+
+on:
+  push:
```

Comment:
Don't you want to run it on `pull_requests` ?

Reply:
I am not sure, why but using `pull_request` will not trigger the integration tests, only `push`.

Reply:
But it works for other workflows: https://github.com/dfinity/repositories-open-to-contributions/blob/master/.github/workflows/check_new_repo.yml#L7

Reply:
I'm still trying to investigate why, but it looks like `pull_request` runs on `master`, so specifying branches-ignore does nothing. Whereas `push` runs on the actual branch and follows the ignore rules.

Reply:
It looks like this is confirmed [here](https://github.com/orgs/community/discussions/26795), so I need to use `push` if I want to filter by branch.